app/main.py

```diff
@@ -1,3 +1,68 @@
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int):
```

Comment:
add type annotations everywhere, and for return values as well

Reply:
> add type annotations everywhere, and for return values as well

def __add__(self, other) -> Distance:

NameError: name 'Distance' is not defined